### ISSUE TITLE
ConEmuExecutablePath empty after partial build

### ISSUE THREAD

**Issue by mstv** [gitextensions/gitextensions#6241]
## Current behaviour

The string `startinfo.ConEmuExecutablePath` is empty after a partial build.

## Steps to reproduce

Hit F5 after some modification to run/debug GE.
Execute some Git command which opens a ConEmu console, e.g. Create Branch.
ConEmuSession.cs(717): `throw new InvalidOperationException("Could not run the console emulator. The path to ConEmu.exe could not be detected.");`
ConEmu is not drawn:
![bad](https://user-images.githubusercontent.com/36601201/52593337-fbe32080-2e48-11e9-868f-4ba3996707e5.png)

## Expected behaviour

Git command is executed and console is shown.

## Did this work in previous version of GitExtensions

I guess no, at least not for 3.x.

## Environment

- Git Extensions 3.1.0
- Build 3d8364575405b4ad3a0c66a38023d2139b25844f (Dirty)
- Git 2.20.1.windows.1
- Microsoft Windows NT 6.1.7601 Service Pack 1
- .NET Framework 4.7.2117.0
- DPI 96dpi (no scaling)

**Comment by vbjay** [comment]
Why we have the build script. Why do you expect a partial build the work?
You HAVE to build a full build to get all binaries.

**Comment by mstv** [comment]
Hold your horses, Jay.
1. The wiki states 'F5' was enough.
2. I had rebuilt the whole solution, fixed the settings dialog, hit `F5` and then wasn't able to create a new branch.

This happens quite often.

**Comment by vbjay** [comment]
CLose solution.   Run git clean -d -x -f.  You now have a clean repo without anything but source controlled files.  Open solution.  Press f5.  Read the build log.  Notice any errors?  If so show them here.

**Comment by RussKie** [comment]
Yes, every now and then GitExtensions/bin/*/ConEmu folder gets cleared, I don't think anyone has investigated it indepth to work out what is causing this

**Comment by RussKie** [comment]
> CLose solution. Run git clean -d -x -f. You now have a clean repo without anything but source controlled files. Open solution. Press f5. Read the build log. Notice any errors? If so show them here.

It doesn't always resolve the issue... It's persistently intermittent or intermittently persistent. 🤷‍♂️ 
I often manually copy ConEmu files from other location

**Comment by vbjay** [comment]
![image](https://user-images.githubusercontent.com/3629489/52595009-e1924e00-2e1a-11e9-9999-41dc0536be00.png)

![image](https://user-images.githubusercontent.com/3629489/52595051-f53db480-2e1a-11e9-91f4-ac43335df133.png)
